samples/security/threat detection/threat-detection-finding-notification-sender.yaml

```diff
@@ -0,0 +1,198 @@
+metadata:
+  version: "1"
+  dependencies:
+    apps:
+      - id: dynatrace.email
+        version: ^1.8.0
+      - id: dynatrace.msteams
+        version: ^2.0.5
+      - id: dynatrace.slack
+        version: ^3.0.4
+  inputs: []
+workflow:
+  title: "Threat Detection Finding - Notification Sender"
+  id: 16515850-2577-4ce7-996b-986125d496ff
```

Comment:
I'm not sure about this - providing an `id´ here is potentially harmful. I know that we want it for calling the sub-workflow, but I doubt that this is sustainable.

Reply:
Clarified internally.